MyMyntra/CollectionViewCell/HomeProductsImagesCollectionViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  HomeProductsImagesCollectionViewCell.swift
+//  MyMyntra
+//
+//  Created by Supraja on 11/02/24.
+//
+
+import UIKit
+
+class HomeProductsImagesCollectionViewCell: UICollectionViewCell {
+    
+    static let id = "HomeProductsImagesCollectionViewCell"
+    
+    @IBOutlet weak var HomeProductsImages: UIImageView!
```

Comment:
- variable name should start with small.
- variable name should be `homeProductsImageView`

Reply:
done